deny.toml

```diff
@@ -52,6 +52,12 @@ skip = [
     {name = "windows_x86_64_msvc" },
     {name = "curve25519-dalek"},
     {name = "hashbrown"},
+    {name = "universal-hash"},
```

Comment:
Does it make sense to deny multiple versions if we usually add a problematic crate to the list of exceptions anyway?

Reply:
I'm not sure what you mean with "problematic crate". But just to tell you my opinion in general, I'm not happy that we have to keep doing this. It kind of defeats the purpose. The only reason I'm putting up with this nonsense is because we want to ensure that scale serialization has a consistent version. To me, everything else kind of doesn't matter.

Reply:
As far as I can see, there is no way to deny multiple versions of one specific crate, so I suppose we don't have a choice and will continue to use `multiple-versions = "deny"`.